lib/__tests__/Autocomplete-test.js

```diff
@@ -65,7 +65,7 @@ describe('Autocomplete acceptance tests', () => {
   it('should highlight top match when `props.value` changes', () => {
     const tree = mount(AutocompleteComponentJSX({}))
     expect(tree.state('highlightedIndex')).toEqual(null)
-    tree.setProps({ value: 'a' })
+    tree.setProps({ value: 'ri' })
```

Comment:
The item we are matching here is `Arizona`.
In the old test we typed `a`, which put the `Arizona` item first and auto-highlighted it.

But if you typed `ri` then still `Arizona` was put as first item, but it was not auto-highlighted, since `ri` does not match the beginning of `Arizona`.

With the changes in this PR, `Arizona` will now correctly be highlighted.